kubeconfig

```diff
@@ -0,0 +1,52 @@
+apiVersion: v1
```

Comment:
This should not be part of the pr

Reply:
@nagendra202 please check and address the above comment at the earliest

Reply:
Not sure how this file has been pushed. Anyhow deleted it.

Reply:
@fbalak could you please review the rest and provide your comments?